docs/writing-docs/mdx.md

```diff
@@ -386,42 +388,23 @@ By applying this pattern with the Controls addon, all anchors will be ignored in
 
 ### The MDX documentation doesn't render in my environment
 
-As Storybook relies on MDX 2 to render documentation, some technical limitations may prevent you from migrating to this version. If that's the case, we've prepared a set of instructions to help you transition to this new version.
+As Storybook relies on the latest version of [MDX](https://mdxjs.com/) to render documentation, some technical limitations may prevent you from migrating to this version. If that's the case, we've prepared a set of instructions to help you transition to this new version.
```

Comment:
Based on what I see here https://github.com/storybookjs/storybook/pull/25303/files#diff-59572a08f167cee7f0025a46c200039f0e97dc011eb05ce32593e5f5d715615cR102 , "latest" is not correct, "MDX 3" would be. When MDX 4 gets released this documentation would be wrong, and upgrading to MDX 4 would most likely be a breaking change to Storybook happening in a major version.

```suggestion
As Storybook relies on [MDX 3](https://mdxjs.com/) to render documentation, some technical limitations may prevent you from migrating to this version. If that's the case, we've prepared a set of instructions to help you transition to this new version.
```

Reply:
@JReinhold I left this temporarily as a reminder for it to be updated in a follow-up pull request, as there's still some work to be done that's not part of this pull request. Appreciate the consideration nonetheless 🙏